src/python/pants/bin/pants_loader.py

```diff
@@ -103,6 +117,7 @@ def main(cls) -> None:
 
         sys.setrecursionlimit(int(os.environ.get(RECURSION_LIMIT, "10000")))
 
+        os.environ["PANTS_SANDBOXER_BINARY_PATH"] = cls.sandboxer_bin() or ""
```

Comment:
This ends up being a good way to communicate this information into the Rust code.